server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/SetMessagesMethodTest.java

```diff
@@ -1366,15 +1366,15 @@ public void event(Event event) {
 
 
         calmlyAwait.atMost(5, TimeUnit.SECONDS).until(() -> events.stream()
-            .anyMatch(event -> isAddedToOutboxEvent(messageId, event)));
+            .anyMatch(event -> isAddedToOutboxEvent(messageId, event, outboxId)));
     }
 
-    private boolean isAddedToOutboxEvent(String messageId, MailboxListener.Event event) {
+    private boolean isAddedToOutboxEvent(String messageId, MailboxListener.Event event, String outboxId) {
```

Comment:
I guess outBoxId should be a mailboxId and not a String.

Reply:
it's parsed from json data which is the output of JMAP so yes, it's string and outboxId is make sense such as it said the flow, sent mailbox is moved to outbox